examples/cookbook/lists/floating_app_bar/analysis_options.yaml

```diff
@@ -1 +1,10 @@
-include: package:pedantic/analysis_options.yaml
\ No newline at end of file
+# Take our settings from the example directory's main analysis_options.yaml
+# file. If necessary for a particular example, this file can also include
+# overrides for individual lints.
+
+include: ../../../analysis_options.yaml
+
+linter:
+  rules:
+    prefer_const_constructors: false
```

Comment:
I'm curious about this elision. I'd prefer to have const constructors in our docs if possible.